src/components/FilterForm/index.jsx

```diff
@@ -0,0 +1,60 @@
+import { Component } from 'react';
+import { withRouter } from 'react-router-dom';
+import MenuItem from '@material-ui/core/MenuItem';
+import TextField from '@material-ui/core/TextField';
+import { withStyles } from '@material-ui/core/styles';
+import { func, string } from 'prop-types';
+import { ASSIGNEE } from '../../utils/constants';
+
+const assigneeList = Object.values(ASSIGNEE);
+
+@withRouter
+@withStyles(theme => ({
+  root: {
+    margin: `0px ${3 * theme.spacing.unit}px`,
+    maxWidth: 360,
+  },
+}))
+export default class FilterForm extends Component {
+  static propTypes = {
+    /**
+     * The current assignee value used to filter items.
+     */
+    assignee: string,
+    /**
+     * The handler for clicking filter items to update the query string.
+     */
+    onFilterItemClick: func,
+  };
+
+  getSelectedAssignee() {
+    const { assignee } = this.props;
+
+    return assigneeList.indexOf(assignee) > -1 ? assignee : ASSIGNEE.UNASSIGNED;
+  }
+
+  handleFilterItemClick = event => {
+    this.props.onFilterItemClick({ assignee: event.target.value });
+  };
+
+  render() {
+    const selectedAssignee = this.getSelectedAssignee();
+    const { classes } = this.props;
+
+    return (
+      <div className={classes.root}>
+        <TextField
+          select
+          label="Assignee"
+          value={selectedAssignee}
+          onChange={this.handleFilterItemClick}>
+          {assigneeList.map(assignee => (
+            <MenuItem key={assignee} value={assignee}>
+              {assignee}
+            </MenuItem>
+          ))}
+        </TextField>
+      </div>
+    );
+  }
+}
```

Comment:
This component simply renders a `TextField`. We don't need a filter component for that. We can just use `TextField` directly inside `TasksTable`.

Reply:
if there is two `TextField`, wdyt? Or should i save it for later?

Reply:
For now let's remove this component and just inline it in `TasksTable`.